screens/AllGiantsScreen/AllGiantsScreen.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { View, Text, ScrollView, TouchableOpacity } from "react-native";
+import Giant from "../../components/Giant/Giant";
+import styles from "./AllGiantsStyles";
+console.disableYellowBox = true;
+import * as Font from "expo-font";
+
+export default class AllGiantsScreen extends React.Component {
+  state = {
+    giants: [
+      {
+        id: 1,
+        name: "Teddy",
+        location: "HØJE TAASTRUP",
+        image: require("../../assets/images/Teddy.png")
```

Comment:
"../../assets/" happens 22 times in the project.
it is too much duplication. 
you could consider defining a function named asset() that concatenats "../../assets/" to a string..